src/System.Text.Json/tests/JsonElementWriteTests.cs

```diff
@@ -901,19 +905,31 @@ public static void WritePropertyOutsideObject(bool skipValidation)
                     {
                         JsonTestHelper.AssertThrows<InvalidOperationException>(
                             ref writer,
-                            (ref Utf8JsonWriter w) => val.WriteProperty(CharLabel, w));
+                            (ref Utf8JsonWriter w) =>
+                            {
+                                w.WritePropertyName(CharLabel);
+                            });
```

Comment:
Since these are being updated already they can be simplified (since the writer is no longer a struct) to just be

```C#
Assert.Throws<InvalidOperationException>(() => w.WritePropertyName(CharLabel));
```

(etc).

Reply:
changed in all places at JsonElementWriteTests.cs